db/migrate/20170706152142_add_category_option_id_to_categories.rb

```diff
@@ -0,0 +1,8 @@
+class AddCategoryOptionIdToCategories < ActiveRecord::Migration
+  def change
+    Category.all.each do |cat|
+      cat.category_option_id = CategoryOption.find_by(name: cat.name).id
```

Comment:
I think this might not work properly with the addition of the `name` method to category. Without the manual addition of `name`, `cat.name` will fetch the field in the database. After adding the new `name` method `cat.name` will fetch the category_option and get it's name which will be null here.

I think the solution is to add `name` to category in another PR after this one has been merged and deployed